meteor/client/lib/EditAttribute.tsx

```diff
@@ -229,7 +229,11 @@ const EditAttributeText = wrapEditAttribute(
 			this.handleEdit(event.target.value)
 		}
 		handleBlur(event) {
-			this.handleUpdate(event.target.value)
+			let value: string = event.target.value
+			if (value) {
+				value = value.trim()
+			}
+			this.handleUpdate(value)
 		}
 		handleEscape(event) {
 			const e = event as KeyboardEvent
```

Comment:
If you should be boyscouting, a good coworker of mine told me this could be changed to something like `const keyboardEvent = event as KeyboardEvent` rather than `e`.

Reply:
Wouldn't it be better the function expect a KeyboardEvent?
```typescript
handleEscape(event: KeyboardEvent): void {
  ...
}